test/unit/SourceLocator/Type/MemoizingSourceLocatorTest.php

```diff
@@ -205,7 +205,7 @@ private function assertMemoization(
                 Reflector $reflector,
                 Identifier $identifier
             ) use (
-                & $fetchedSymbolsCount
+                &$fetchedSymbolsCount
```

Comment:
This is potentially caused by `doctrine/coding-standard:6.0.0` depending on PSR-12 rules that were changed in upstream phpcs. Upgrading to `doctrine/coding-standard:7.0.2` should fix these, in theory.

Reply:
@kukulich told me that he'll be updating BetterRefleciton to `doctrine/coding-standard:7.x` so I'll leave it up to him :)